README.md

```diff
@@ -114,6 +118,23 @@ The app can then be accessed at `localhost:5001`.
 
 Changes made to the source code will not affect the production application server.
 
+## Running the container on Azure
+
+We can run the To-do app on Azure (at tedwhi-todo-app.azurewebsites.net) by deploying the container on an Azure App Service.
+
+First we must deploy the container image to Docker Hub, this can all be done from the CLI:
+```bash
+docker login
+docker build --target production --tag tedwhi/todo-app:prod .
```

Comment:
Wasn't quite sure what to do here, are we expecting a "new user" to
- be logged into their own docker hub account and publish to it (which would require changing the App Service config), or
- to be logged into my own docker hub account and publish to it, or
- be able publish to `tedwhi/` on Docker hub

or are we just accepting that this isn't a fully realistic scenario for team collaboration?

Reply:
Some version of 2 & 3:
* Ideally we'd have some "organisation" set up which can grant users permission to push to a particular repo (e.g. [DockerHub has organisations](https://docs.docker.com/admin/organization/orgs/), but we don't use them here simply because that costs)
* A workaround that teams do often use is to have a "team user" that people have shared access to (e.g. we have used `corndeldevopscourse` in the past in this way as a team)
* And an alternative is, once we have a pipeline, to have the only user with permission be the one the pipeline uses to sign in, and others should push only through the pipeline
  * (In practice, we'd likely want to combine that with the "organisation" though, so that we can also grant exceptional access outside the pipeline)

Reply:
Thanks for the explanation :)))